bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Please right a better title e.g. "biorbd #1 - Rototrans".</p><p>Put this tutorial into a subfolder call biorbd because it is specific to biorbd, we will create another folder for bioptim specific tutorials.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>